### DISCUSSION TITLE
Policy update violation

### DISCUSSION THREAD

**Discussion by navi86** [opentofu/opentofu#1955] [General]
Base on this topic, 

https://github.com/opentofu/registry/pull/824

i wanna make sure that i understand correctly, that opentofy violated Mozilla Public License and maintained by Linux Foundation so it should change license to other type of license.
Please give explanation @cam72cam

**Comment by kvendingoldo** [comment]
Yes. For example, the request about exclusion from CNCF is here https://github.com/cncf/sandbox/issues/81; The same should be done for Linux Foundation

**Comment by gecube** [comment]
In Slack as well COMPLETE Silence https://cloud-native.slack.com/archives/C05PXGAB05R/p1724936434461019

It is not the behaviour that we are expecting from LF and/or CNCF project.

**Comment by gecube** [comment]
@ollevche @Yantrio

**Comment by cam72cam** [comment]
In discussing with the Linux Foundation, we understand this is a complex issue with many countries imposing sanctions on Russia and we do not have the community resources to figure this all out right now. Anyone in any geography could provide a similar service locally.

**Reply by kvendingoldo** [reply]
Which community resources do you need? We have a lot of people who participated in last discussion who are ready to help.

**Reply by cam72cam** [reply]
After consulting with the Linux Foundation we are led to believe that we do not have the resources to determine if our hosted services are exempt from US and EU sanctions. Our next steps are to clarify our policy and will continue to coordinate with the Linux Foundation before taking additional actions.

The OpenTofu Registry is open source under the MPL license and only uses the GitHub API, which makes it open for anyone with access to GitHub to offer a registry service.

**Reply by kvendingoldo** [reply]
> The OpenTofu Registry is open source under the MPL license and only uses the GitHub API, which makes it open for anyone with access to GitHub to offer a registry service.

Agree, community can make a project under OpenTofu for users from Russia, China, Lebanon, etc. WDYT? 

Btw. if you say that it open, does it mean that providers will be returned to registry?

**Reply by gecube** [reply]
@kvendingoldo you may proceed in Slack of the opentofu community

https://opentofucommunity.slack.com/archives/C05QF5HA0BH/p1725447796135029

there is the same discussion with the same outcomes.

**Comment by kvendingoldo** [comment]
Bumped the thread: https://github.com/opentofu/registry/issues/937

**Reply by gecube** [reply]
+1
